src/routes/api/articleRoutes.js

```diff
@@ -1,9 +1,43 @@
 import express from 'express'
-
+import ArticleController from "../../controllers/articleController"
+import CommentController from '../../controllers/commentController'
+import multer from "multer"
+import { fileFilter } from '../../helpers/fileFilter'
 const route = express.Router()
+const storageFile = multer.diskStorage({}) // take none because image will be upload on cloudinary no locally
+const upload = multer({storage: storageFile, file: fileFilter})
+
+route.post('/create', upload.single("image"), async (req, res, next) => {
+    console.log("route run")
+    await ArticleController.createArticle(req, res)
+})
```

Comment:
Replace this with `route.post("/create", upload.single("image"), ArticleController.createArticle)` and implement the same on other routes

Reply:
changes made successful